LabTools/Display/PlotDisplayWindow.py

```diff
@@ -39,14 +39,25 @@
     from PyQt4.QtCore import Qt, QRect, QRectF
     from PyQt4.QtCore import SIGNAL
 
-
-import QtTools
-
-
-from mplZoomWidget import MatplotlibZoomWidget
-import ui_plotdisplaywindow
-
-from PlotPreferences import marker_set, line_set, color_blind_friendly_colors
+try:
+    import QtTools
+except:
+    from . import QtTools
+
+
+try:
+    from mplZoomWidget import MatplotlibZoomWidget
+except:
+    from . import mplZoomWidget
+    MatplotlibZoomWidget = mplZoomWidget.MatplotlibWidget
+try:
+    import ui_plotdisplaywindow
+except:
+    from . import ui_plotdisplaywindow
+try:
+    from PlotPreferences import marker_set, line_set, color_blind_friendly_colors
+except:
+    from .PlotPreferences import marker_set, line_set, color_blind_friendly_colors
```

Comment:
I think we could try with the full path `Labtools.Display.<name of module>` instead of the `try except` statements. This is not an urgent change.

Reply:
This could potentially work, especially since I added to LabGui.py to change the cwd on launch. My only concern with removing try/except statements fully would be disallowing these scripts to run on their own for testing purposes, using if __name__ == "__main__": at the end of each one. This could potentially be corrected by adding a functionality for end-user testing, which would have cwd be set to LabGUI root, and then executing any user test script (a GUI module for this with script selector/writer and console output would be very doable, I will add it to my todo list!)